extensions/adobe/experience/decisioning/proposition-detail.example.1.json

```diff
@@ -0,0 +1,23 @@
+{
+  "xdm:activity": {
+    "@id": "xcore:offer-activity:ebc48132c26ccfc",
+    "xdm:name": "Tutorial videos to watch",
+    "xdm:startDate": "2018-10-13T05:59:18.914Z",
+    "xdm:endDate": "2018-12-27T05:59:18.914Z",
+    "xdm:fallback": "xcore:fallback-offer:e91ce7243fd8c2a",
+    "xdm:placement": "xcore:offer-placement:e91a84cc83ad12f",
+    "xdm:filter": "xcore:offer-filter:e9201a6650bb7e3"
```

Comment:
"xdm:filter" is not in the activity schema definition.

Reply:
Can be removed from the example for now